### ISSUE TITLE
Converting Unbound to JSON

### ISSUE THREAD

**Issue by barryanders** [acoll/bible-cli#1]
Hello, I installed the necessary node modules and ran `mocha library.test.js`

The first time, I got this:

```
 Library
    .translations(fn)
      ✓ should return the list of translations 
      1) should return the list of translations
    .convert(bookfile, biblefile, fn)
      2) should convert the unbound format to json


  1 passing (10ms)
  2 failing

  1) Library .translations(fn) should return the list of translations:
     Uncaught AssertionError: 0 == 2
      at /Applications/MAMP/htdocs/bible-cli/test/library.test.js:10:12
      at /Applications/MAMP/htdocs/bible-cli/lib/library.js:15:3
      at ReaddirReq.Req.done (/usr/local/lib/node_modules/mocha/node_modules/glob/node_modules/graceful-fs/graceful-fs.js:143:5)
      at ReaddirReq.done (/usr/local/lib/node_modules/mocha/node_modules/glob/node_modules/graceful-fs/graceful-fs.js:90:22)
      at Object.oncomplete (fs.js:107:15)

  2) Library .convert(bookfile, biblefile, fn) should convert the unbound format to json:
     Uncaught Error: ENOENT, open './raw-translations/asv/book_names.txt'
```

So I moved the translations and raw-translations into the test folder and ran it again to get this:

```
 Library
    .translations(fn)
      ✓ should return the list of translations 
    .convert(bookfile, biblefile, fn)
      ✓ should convert the unbound format to json (539ms)


  2 passing (548ms)
```

Which appeared to pass, but I couldn't find the output file. So I ran it again and got this error:

```
 Library
    .translations(fn)
      ✓ should return the list of translations 
      1) should return the list of translations
    .convert(bookfile, biblefile, fn)
      ✓ should convert the unbound format to json (493ms)


  2 passing (503ms)
  1 failing

  1) Library .translations(fn) should return the list of translations:
     Uncaught AssertionError: 1 == 2
      at /Applications/MAMP/htdocs/bible-cli/test/library.test.js:10:12
      at /Applications/MAMP/htdocs/bible-cli/lib/library.js:15:3
      at ReaddirReq.Req.done (/usr/local/lib/node_modules/mocha/node_modules/glob/node_modules/graceful-fs/graceful-fs.js:143:5)
      at ReaddirReq.done (/usr/local/lib/node_modules/mocha/node_modules/glob/node_modules/graceful-fs/graceful-fs.js:90:22)
      at Object.oncomplete (fs.js:107:15)
```

So I guess my question is... what am I doing wrong? I want to convert unbound to json just like your ASV example.

**Comment by barryanders** [comment]
I figured it out. Apparently, all I had to do was run `mocha` from the root folder. I will likely implement this into the Bible project I'm working on. That conversion to JSON is handy.

**Comment by acoll** [comment]
Sorry. I intended to get back to you when i get home from work today. Whats
the project you are working on? Do you have it on github? I'm not sure of
the current state of this project. It was back when i first started using
nodejs and was playing around with the REPL to build a command line
interface to lookup verses.

**Comment by barryanders** [comment]
I have a repository I have been working on that outputs the JSON Bible you have in this project to static HTML. I was trying to figure out how to use this to convert other Unbound Bibles to that same JSON file. I'm not sure if I completely understand how it works or not. I thought I had it figured out but now I'm not sure.

**Comment by acoll** [comment]
Yea i'm sure the code is pretty ugly. I was just learning node / js.
Basically i try to parse the unbound format to put the whole bible into a
json block that is something like

{
   Genesis: {
      1: {
         1: 'In the beginning God created the heavens and the earth.',
         2: '......'

and so on.

And then lookups would be really easy:

bible['Genesis'][1][1] would return 'In the beginning...'

I was doing a weird split thing on a tab character. If i had to redo it i'd
prob use a regex. The book names are in their own file with the number.

Whats the repository url?

Do you have another unbound version that  follows the same format (having a
book file with the number to book name and a raw utf8 file)?

Whats your repository?

**Comment by barryanders** [comment]
This is my Repo: https://github.com/barryanders/Ezra.js

I haven't documented how to use it, but basically, it currently passes a JSON file like this one below and generates an HTML file for each book, chapter, and verse:
https://raw.githubusercontent.com/honza/bibles/master/ESV/ESV.json

I was able to use your ASV.json because it is almost exactly the same as the ESV one. I just removed the stuff at the top (version and books) and then it processed. I am currently under the impression that I can download [Unbound Bibles](http://unbound.biola.edu/index.cfm?method=downloads.showDownloadMain) that can then be turned into this same JSON file by using your bible-cli. I'm just not sure how to do that.

I don't currently have any raw Unbound files other than the ones included with bible-cli.
